README.mdown

```diff
@@ -4,196 +4,42 @@ CakePHP Documentation
-    // re-run the build command
-    docker build -t cakephp/docs .
-
-Now that the image is built, you can run all the commands to build the docs:
-
-    # To build the html
-    cd /path/to/your/local/docs
-    docker run -it --rm -v $(pwd):/data cakephp/docs make html
-
-    # To build the epub
-    cd /path/to/your/local/docs
-    docker run -it --rm -v $(pwd):/data cakephp/docs make epub
-
-    # To build the latex
-    cd /path/to/your/local/docs
-    docker run -it --rm -v $(pwd):/data cakephp/docs make latex
-
-    # To build the pdf
-    cd /path/to/your/local/docs
-    docker run -it --rm -v $(pwd):/data cakephp/docs make pdf
-
-All the commands below will create and start containers and build the docs in
-the `build` folder. The `--rm` flag will delete the container after run.
-
-Build the Documentation Manually
---------------------------------
-
-### Installing the needed Packages ###
-
-To build the documentation you'll need the following for linux/OS X:
-
-* Make
-* Python
-* Sphinx 1.2.* (currently the make commands will not work with 1.3.* versions
-  and up)
-* PhpDomain for sphinx
-
-You can install sphinx using:
-
-    pip install sphinx==1.2.3
-
-You can install the phpdomain using:
-
-    pip install sphinxcontrib-phpdomain
-
-*To run the pip command, python-pip package must be previously installed.*
-
-### Building the Documentation ###
-
-After installing the required packages, you can build the documentation using
-`make`.
-
-    # Create all the HTML docs. Including all the languages.
-    make html
-
-    # Create just the English HTML docs.
-    make html-en
-
-    # Create all the EPUB (e-book) docs.
-    make epub
-
-    # Create just the English EPUB docs.
-    make epub-en
-
-    # Populate the search index
-    make populate-index
-
-This will generate all the documentation in an HTML form. Other output such as
-'htmlhelp' are not fully complete at this time.
-
-After making changes to the documentation, you can build the HTML version of the
-docs by using `make html` again.  This will build only the HTML files that have
-had changes made to them.
-
-### Building the PDF ###
-
-Building the PDF is a non-trivial task.
-
-1. Install LaTeX - This varies by distribution/OS so refer to your package
-   manager. You should install the full LaTeX package. The basic one requires
-   any additional packages to be installed with `tlmgr`
-2. Run `make latex-en`.
-3. Run `make pdf-en`.
-
-At this point the completed PDF should be in build/latex/en/CakePHPCookbook.pdf.
-
-Contributing
-------------
-
-There are currently a number of outstanding issues that need to be addressed.
-We've tried to flag these with `.. todo::` where possible. To see all the
-outstanding todo's add the following to your `config/all.py`
+There are currently a number of outstanding issues that need to be addressed. We've tried to flag these with `.. todo::` where possible. To see all the outstanding todo's add the following to your `config/all.py`
```

Comment:
Have I missed something? Why are those lines not broken down into many lines as we normally do?
Some more below.